Moq.AutoMock/AutoMocker.cs

```diff
@@ -279,8 +277,9 @@ public void Use<TService>(Expression<Func<TService, bool>> setup)
         /// </summary>
         /// <typeparam name="TService">The class or interface to search on</typeparam>
         /// <returns>The object that implements TService</returns>
-        public TService? Get<TService>() where TService : class
-            => Get(typeof(TService)) is TService service ? service : null;
+        [return:MaybeNull]
+        public TService Get<TService>()
+            => Get(typeof(TService)) is TService service ? service : default;

```

Comment:
This is fixed as agreed in my PR.

Reply:
Opps didn't mean to leave this in. Reverted.